src/test/kotlin/uk/gov/justice/digital/hmpps/hmppstiertodeliusupdate/integration/endtoend/TierTest.kt

```diff
@@ -56,7 +57,7 @@ internal class TierTest : MockedEndpointsTestBase() {
     awsSqsClient.sendMessage(queue, tierUpdateMessage())
 
     await untilCallTo { getNumberOfMessagesCurrentlyOnQueue() } matches { it == 0 }
-    (await withPollInterval ONE_HUNDRED_MILLISECONDS).ignoreException(IllegalArgumentException::class)
+    (await withPollInterval ONE_MILLISECOND).ignoreException(IllegalArgumentException::class)
```

Comment:
it seems as if one hundred milliseconds is too long to verify the call to the API? Within that one hundred millisecond interval and the until asserted the message already went on the dlq